python/LHCHCGModels.py

```diff
@@ -384,18 +392,18 @@ def setup(self):
             for d in SM_HIGG_DECAYS: 
                 self.modelBuilder.factory_('HiggsDecayWidth_UncertaintyScaling_%s[1.0]' % d)
         # get VBF, tHq, tHW, ggZH cross section
-        self.SMH.makeScaling('qqH', CW='kappa_W', CZ='kappa_Z')
-        self.SMH.makeScaling("tHq", CW='kappa_W', Ctop="kappa_t")
-        self.SMH.makeScaling("tHW", CW='kappa_W', Ctop="kappa_t")
+        self.SMH.makeScaling('qqH', CW=self.kappa_W, CZ=self.kappa_Z)
+        self.SMH.makeScaling("tHq", CW=self.kappa_W, C, Ctop="kappa_t")
+        self.SMH.makeScaling("tHW", CW=self.kappa_W, C, Ctop="kappa_t")
```

Comment:
Looks like a stray `, C,` here - I guess this stops the physics model from working at all?

Reply:
Sorry.  Fixed.